src/GeneralStateTestsFiller/stExample/balanceFiller.json

```diff
@@ -0,0 +1,93 @@
+{
+    "balance" : {
+        "_info" : {
+            "comment" : "A test for balance opcode result"
+        },
+        "env" : {
+            "//comment" : "A blockinfo section (obsolite). Do not change. (only coinbase and gasLimit could be changed)",
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "1000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "//comment" : "Expect section is the post condition for the transaction result",
+        "expect" : [
+            {
+                "//comment" : "indexes are used to specify post condition for a given transaction range.",
+                "//comment" : "transaction section describe multiple transaction which will be executed on pre state",
+                "indexes" : {
+                    "//comment" : "-1 for transaction with any data from this field. or a particular index, or range [0, 1, 2]",
+                    "data" : -1,
+                    "gas" : 0,
+                    "value" : -1
+                },
+                "//comment" : "hard fork order: Frontier, Homestead, EIP150, EIP155, Byzantium, Constantinople, ConstantinopleFix",
+                "network" : [">=ConstantinopleFix"],
```

Comment:
Is this supposed to be `[>=Istanbul]`?

Reply:
With Retesteth config files you can specify your own Fork names and fork order